### DISCUSSION TITLE
I hope to add a temporary capture clip function and customize the shortcut key

### DISCUSSION THREAD

**Discussion by tiger7651** [hluk/CopyQ#2545] [Ideas]
I hope to add a temporary capture clip function and customize the shortcut key, that is, when the store clipboard (Preferences\General) is disabled, you can copy items to copyQ through the custom shortcut key

Or it's called forced capture function

This feature is available in Clipdiary, which is very convenient
![image](https://github.com/hluk/CopyQ/assets/150779117/e660efdb-8fb1-43c9-80f5-e829a66f7ce6)

**Comment by GFDGIT** [comment]
```
[Command]
Name=Force Capture
Command="
    copyq:
    copy()
    if (monitoring())
        abort()
    var item = {}
    for (const format of clipboardFormatsToSave()) {
         var data = clipboard(format)
         if (data.length)
             item[format] = data
    }
    add(item)"
IsGlobalShortcut=true
Icon=\xf0c5
GlobalShortcut=alt+c
```

**Reply by tiger7651** [reply]
Thank you! If set in this way, the code is invalid.
![image](https://github.com/hluk/CopyQ/assets/150779117/844e918d-e587-4f80-8c89-b9b286a4f5f4)

But this setting is effective.
![image](https://github.com/hluk/CopyQ/assets/150779117/1357f735-204f-4a18-a46b-1a5585bbc397)

May I ask how to automatically switch to Disable clipboard storage after enabling copyQ
![image](https://github.com/hluk/CopyQ/assets/150779117/a1bc993b-ff8f-4d87-828e-9daae8ace745)

**Reply by GFDGIT** [reply]
> But this setting is effective. 

Because this method provides shortcut, I prioritized it.

> after enabling copyQ

Are you saying restart copyq?
If yes, this script will be helpful: [remeber-clipboard-storing-state](https://github.com/hluk/copyq-commands/blob/master/Scripts/remeber-clipboard-storing-state.ini)

**Reply by tiger7651** [reply]
Yes, thank you
